data_generator.py

```diff
@@ -10,14 +10,20 @@
 
 
 class DataGenerator(Sequence):
-    def __init__(self, list_indices, path_input, batch_size, features, tile_size, num_classes, shuffle=True,
-                 png_form=False):
+    def __init__(self, list_indices, path_input, batch_size, features, tile_size, num_classes, product_level,
+                 shuffle=True, png_form=False):
         """ Initialization """
         self.path = path_input
-        self.stds = [0.00085, 0.04, 0.037, 0.035, 0.034, 0.035, 0.033, 0.035, 0.034, 0.054, 0.025, 0.021, 0.0083]
-        self.means = [0.0009, 0.02, 0.02, 0.02, 0.02, 0.041, 0.047, 0.045, 0.06, 0.03, 0.03, 0.022, 0.015]
-        self.min_v = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-        self.max_v = [0.0039, 0.298, 0.282, 0.266, 0.255, 0.252, 0.246, 0.241, 0.241, 0.249, 0.219, 0.221, 0.076]
+        if product_level == "L2A":
```

Comment:
These should also be read from a file, instead of being hard-coded here.

Reply:
Same as bands, we need to discuss it